src/pybind/mgr/dashboard/model/feedback.py

```diff
@@ -0,0 +1,56 @@
+# # -*- coding: utf-8 -*-
+from enum import Enum
+
+class Feedback:
+    project_id: int
+    tracker_id: int
+    subject: str
+    description: str
+    severity: str
+    status: int
+
+    class Project(Enum):
+        dashboard = 46
+        block = 9 #rbd
+        object = 10 # rgw
+        file_system = 13 # cephfs
```

Comment:
For this type of classes I would use dataclasses https://docs.python.org/3/library/dataclasses.html.